backend/src/Edna/Web/Server.hs

```diff
@@ -111,4 +113,7 @@ edna = do
       -- No dev logging, no docs
       | otherwise = serve ednaAPI server
 
-  serveWeb listenAddr loggingConfig app
+  _ <- register ghcMetrics
+  let promMiddleware = prometheus $ PrometheusSettings ["metrics"] True False
```

Comment:
Can you please use record syntax to define `PrometheusSettings`? Without explicit record fields it's hard to understand the meaning of 3 values passed to the `PrometheusSettings` constructor.

Reply:
To be fair, it’s not really because of the record syntax, rather about settings themselves :). With record syntax it would be `prometheusEndPoint`, `prometheusInstrumentApp`, `prometheusInstrumentPrometheus`, which still do not explain anything, IMHO 😆, since they managed to make this configuration type pretty confusing.

I guess what could help a bit is if we used `def` instead to simply hide all this complexity and never think about it again 🙂, and then only use record syntax to override `prometheusInstrumentPrometheus` to `False` (and leave a comment explaining why we do this).

Reply:
Well, now it's
```
  let promMiddleware = prometheus $ PrometheusSettings
        { prometheusEndPoint = ["metrics"]
        , prometheusInstrumentApp = True
        , prometheusInstrumentPrometheus = False
        }
```

so I at least more or less understand what `["metrics"]` is.

Reply:
> I guess what could help a bit is if we used def instead to simply hide all this complexity and never think about it again slightly_smiling_face, and then only use record syntax to override prometheusInstrumentPrometheus to False (and leave a comment explaining why we do this).

@worm2fed can you do it please?

Reply:
yes

Reply:
oh, there are def definition in package already